components/dashboard/src/components/SelectIDEComponent.tsx

```diff
@@ -141,17 +141,12 @@ export default function SelectIDEComponent({
         if (shouldShowDeprecationNotice) {
```

Comment:
I guess we can remove this now since `useAllowedWorkspaceEditorsMemo` should not return the IDE anymore?